.github/workflows/deployment.yml

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) 2023 Files Community
+# Licensed under the MIT License. See the LICENSE.
+
+# Abstract:
+#   This workflow is triggered manually when the maintainer wants
+#   to deploy Preview or Stable to the Azure Blob Storage.
+
+name: Files Deployment
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Select a branch'
+        required: false
+        default: 'Preview'
```

Comment:
Can we force an option to be selected?

Reply:
You mean that you'd like to be forced to select at any time?

`required: true` can do it if so.

Reply:
I meant to have nothing selected by default

Reply:
You gonna need to select every time. Is that ok?

Reply:
That's what I'm looking for, thanks!

Reply:
> Is that ok?

I meant that you are fine with that? You gonna need to select every time. you trigger that pipeline mostly for the preview package. Isn't that gonna be troublesome?

Reply:
Yes